jobs/tsa/templates/tsa_ctl.erb

```diff
@@ -51,10 +56,15 @@ EOF
 
     # TODO: TSA should support multiple ATCs
     exec chpst -u vcap:vcap /var/vcap/packages/tsa/bin/tsa \
+      <% if if_link("atc") %>
```

Comment:
This looks a little off. `if_link` is supposed to take a block to execute. The `else` makes this tricky but you could just pull these conditionals out and assign a local variable above:

```ruby
atc_url = nil

if_link("atc") do |atc|
  atc.instances.each do |instance|
    atc_url = "http://#{instance.address}:#{atc.p("bind_port")}"
  end
end

atc_url ||= p("atc.url")
```